core/src/com/unciv/logic/automation/civilization/ReligionAutomation.kt

```diff
@@ -30,11 +30,15 @@ object ReligionAutomation {
             buyGreatProphetInAnyCity(civInfo)
             return
         }
+        
+        if (civInfo.religionManager.remainingFoundableReligions() == 0 && civInfo.religionManager.religionState != ReligionState.FoundingReligion) {
```

Comment:
We should change this to "religionState == Pantheon", because if we *already have* a religion but there are 0 foundable, we still want a new great prophet because that gives us another belief!

Reply:
Are you sure about this? The AI stops buying great persons in certain cases with that change.

Reply:
Nevermind, I made a typo